src/load/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+1.4.0
+++++++
+* Add support for multi-region load test configuration. Multi-region load test configuration can be set using `--regionwise-engines` argument in 'az load test create' and 'az load test update' commands. Multi-region load test configuration set in YAML config file under key `regionalLoadTestConfig` will also be honoured.
```

Comment:
how do we deal with conflict if both are set? which takes precedence?

Reply:
CLI arguments take precedence, however in this case since engine-instances count need to correlate with the regional distribution, we first evaluate the final data for both engine-instances and regional load and then validate

ex:
I have set engines instances 2 in config file but regional load is set in CLI arguments eastus=1 westus2=1, this is valid.
I have set eastus=2 westus2=2 in CLI arguments and engine instances 4 in the config file, this is valid
I have set eastus=2 westus2=3 in CLI arguments , engine instances as 5 in CLI arguments , but engine instances is 4 in config file , by precedence rule engine instances will evaluate to 5 and count matches for regional load, this is valid

Reply:
The same pattern for the regional load. Point to note, we don't combine the data from CLI arguments and config file. So the complete regional load object in the payload will be overwritten
ex:
I set CLI arguments eastus=1 westus2=2 and in config file westus3=2, the payload will not contain the data which came from config file